src/coreclr/src/pal/inc/pal.h

```diff
@@ -3855,7 +3855,6 @@ PAL_GetCurrentThreadAffinitySet(SIZE_T size, UINT_PTR* data);
 #define log2          PAL_log2
 #define log10         PAL_log10
 #define pow           PAL_pow
-#define scalbn        PAL_scalbn
```

Comment:
Could you please also delete `src\coreclr\src\pal\tests\palsuite\c_runtime\scalbn` and ``src\coreclr\src\pal\tests\palsuite\c_runtime\scalbnf`

Reply:
The output of `git grep scalbn -- :/src/coreclr/src/pal` also shows related entries in:

* `src/coreclr/src/pal/tests/palsuite/CMakeLists.txt`
* `src/coreclr/src/pal/tests/palsuite/compilableTests.txt`
* `src/coreclr/src/pal/tests/palsuite/paltestlist.txt`